lib/job_aggregator.rb

```diff
@@ -2,16 +2,29 @@ class JobAggregator
   def initialize
     @ignore_link_text ||= []
     @url ||= ''
+    @num_pages = 1
```

Comment:
We now default to a single page, could just as easily default to 0, but this feels a little safer.